### DISCUSSION TITLE
Column text alignment and use of custom class

### DISCUSSION THREAD

**Discussion by jonathanoeijoeng** [rappasoft/laravel-livewire-tables#715] [Q&A]
Hi,

This is awesome and make my life much easier to build a table.
I want to ask 2 questions:

- How to align center or right for certain column? I cannot find it in available method in column
- I made custom class to change background color but seems not working when I add it to theadattribute? Is it possible to use custom class in style.css?

Thanks so much

**Comment by edikurniawan-dev** [comment]
1. to align center or right i use custom row in version 1, like this `<x-livewire-tables::table.cell class="text-center">`
2. to change background color, first i publish view of table `php artisan vendor:publish --provider="Rappasoft\LaravelLivewireTables\LaravelLivewireTablesServiceProvider" --tag=livewire-tables-views` and you can change class in the th or another attribute

hope it's answered
